scipy/interpolate/tests/test_interpnd.py

```diff
@@ -62,6 +62,41 @@ def test_tri_input(self):
         yi = interpnd.LinearNDInterpolator(tri, y)(x)
         assert_almost_equal(y, yi)
 
+    def test_tri2_input(self):
+        # be sur that Delaunay triangulation is not recompute.
+        #  the fake tri have a hole in the middle if recompute it will be "closed",
+        #  if not center get NAN value.
+        class FakeDelaunay(qhull.Delaunay):
+            def __init__(self):
+                self._points = np.array([(0,0), (0,1), (0,2),
+                                         (1,0), (1,1), (1,2),
+                                         (2,0), (2,1), (2,2)], dtype=np.float64)
+                self.simplices = np.array([(0,1,3),(1,2,5),(3,7,6), (5,8,7)],
+                                          dtype=np.int32)
+                self.neighbors = np.array([(-1,-1,-1),
+                                           (-1,-1,-1),
+                                           (-1,-1,-1),
+                                           (-1,-1,-1)], dtype=np.int32)
+                self.ndim = self._points.shape[1]
+                self.npoints = self._points.shape[0]
+                self.min_bound = self._points.min(axis=0)
+                self.max_bound = self._points.max(axis=0)
+                self.equations = np.zeros((9, 4), dtype=np.double)
+                self.paraboloid_scale = 1
+                self.paraboloid_shift = 0
+                self._transform = None
```

Comment:
Is there a reason for not just using `Delaunay` directly, this seems potentially fragile if the internal implementation of `Delaunay` changes?

Reply:
Yes I have a reason, but may be not a good one.
There is already a test that create a Delaunay triangulation.
In that case I want to be sure that this triangulation is not recompute. So I try to give the interpolation a "FakeDelaunay" that do not have same result if it is recompute. 

But I can perfectly understand your point. 
Tell me what you think. I can only keep the `assert interpolator.tri is tri` and possibly "merge" this assert on the `test_tri2_input` test.